schema/2.0.0/corpora_schema.md

```diff
@@ -39,6 +39,12 @@ This document is organized by:
 
 * **AnnData** - The canonical data format for the cellxgene Data Portal is HDF5-backed [AnnData](https://anndata.readthedocs.io/en/latest) as written by version 0.7 of the anndata library.  Part of the rationale for selecting this format is to allow cellxgene to access both the data and metadata within a single file. The schema requirements and definitions for the AnnData `X`, `uns`, `obs`, and `obsm` attributes are described below.
 
+* **Organisms**. Datasets MUST contain either Human or Mouse data. No other organisms are accepted by the cellxgene Data Portal. 
+
+* **Name Collisions in Metadata**. If a metadata key exists in the original dataset with the exact name required by the schema but with values that cannot be accurately mapped to the schema, `"key"` MUST be renamed to `"key_original"`. For example, if there is a `"cell_type"` in the original dataset with values that cannot be mapped, then it MUST be renamed to `"cell_type_original"`. 
```

Comment:
Why have an opinion about what the non-schema field is called?
AnnData enforces unique obs keys, and cxg enforces the schema fields meet requirements, so I am not sure what needs to be mentioned here.

Reply:
> AnnData enforces unique obs keys

Just validated with Ambrose, AnnData does not enforce uniqueness.

Reply:
I'm happy to simplify:

1. Metadata keys specified in the cellxgene schema are "reserved" and MUST be unique.  Validation MUST fail if a duplicate of a schema key is detected. Validation will NOT attempt to choose between the duplicate keys and rename one.
2. I defer to @ambrosejcarr where he wants to request that curators rename collisions in a specific way for tracking or visibility. It was done in the past, because it was automated for apply which is now deprecated -  https://github.com/chanzuckerberg/single-cell-curation/blob/f54de6f710fbe7d66de76bec0249931b87e4930f/cellxgene_schema_cli/cellxgene_schema/remix.py#L59

Reply:
> I defer to @ambrosejcarr where he wants to request that curators rename collisions in a specific way for tracking or visibility. It was done in the past, because it was automated for apply which is now deprecated -

I do not think we should specify collisions be renamed in a particular way; it seems impossible to enforce. *"cell_type_original"? Gross, my column was called "cell_type**s**"... promise*.

Reply:
Apologies for my misunderstandings - thanks for straightening me out.
>Metadata keys specified in the cellxgene schema are "reserved" and MUST be unique. Validation MUST fail if a duplicate of a schema key is detected. Validation will NOT attempt to choose between the duplicate keys and rename one.

I like this.

Reply:
Addressed in https://github.com/chanzuckerberg/single-cell-curation/pull/67/commits/d3d66a05fd26d5803004e7adb59bf73b021f39e6